index.js

```diff
@@ -1,4 +1,4 @@
 // https://regex101.com/r/SQrOlx/14
 export default function issueRegex() {
-	return /(?<![\w./-])(?:(?<organization>\w[\w.-]{0,38})\/(?<repository>\w[\w.-]{0,99}))?#(?<issueNumber>[1-9]\d{0,99})\b/g;
+	return /(?<!\w)(?:(?<organization>[a-z\d](?:[a-z\d-]{0,37}[a-z\d])?)\/(?<repository>[\w.-]{1,100}))?(?<!(?:\/\.{1,2}))#(?<issueNumber>[1-9]\d{0,99})\b/gi;
```

Comment:
Let's limit the issue number length to 10. There will never be 9 billion issues in a repo ever.